src/uct/api/uct.h

```diff
@@ -1159,7 +1174,32 @@ UCT_INLINE_API ssize_t uct_ep_put_bcopy(uct_ep_h ep, uct_pack_callback_t pack_cb
 
 /**
  * @ingroup UCT_RMA
- * @brief
+ * @brief Provides RDMA PUT operation
+ *
+ * The input data in @a iov array of @ref ::uct_iov_t structures sent to remote
+ * address ("gather output"). Buffers in @a iov are processed in array order.
+ * This means that the function complete iov[0] before proceeding to
+ * iov[1], and so on.
+ *
+ *
+ * @param [in] ep          Destination endpoint handle.
+ * @param [in] iov         Points to an array of @ref ::uct_iov_t structures.
+ *                         The @a iov pointer must be valid address of an array
+ *                         of @ref ::uct_iov_t structures. A particular structure
+ *                         pointer must be valid address. NULL terminated pointer
+ *                         is not required.
+ * @param [in] size        Size of the @a iov data @ref ::uct_iov_t structures
+ *                         array. If @a size is zero, no data operation provided.
+ *                         @a size is limited by @ref uct_iface_attr_cap_put_max_iov
+ *                         "uct_iface_attr::cap::put::max_iov"
+ * @param [in] remote_addr Remote address to place the @a iov data.
+ * @param [in] rkey        Remote key descriptor provided by @ref ::uct_rkey_unpack
+ * @param [in] comp        Completion handle as defined by @ref ::uct_completion_t.
+ *
+ * @return UCS_INPROGRESS  Some communication operations are still in progress.
+ *                         If non-NULL @a comp is provided, it will be updated
+ *                         upon completion of these operations.
+ *
  */
 UCT_INLINE_API ucs_status_t uct_ep_put_zcopy(uct_ep_h ep,
```

Comment:
@yosefe Need to carefully check this documentation. It looks good in html but I suspect it isn't correct.